frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterDropdown.tsx

```diff
@@ -12,13 +13,7 @@ export function ActionInfo({ item }: RenderInfoProps): JSX.Element {
     return (
         <>
             <AimOutlined /> Actions
-            <Link
-                to={`/action/${item.id}#backTo=Insights&backToURL=${encodeURIComponent(
-                    window.location.pathname + window.location.search
-                )}`}
-                style={{ float: 'right' }}
-                tabIndex={-1}
-            >
+            <Link to={item.id ? urls.action(item.id) : urls.createAction()} style={{ float: 'right' }} tabIndex={-1}>
```

Comment:
Shouldn't we just say that there's no such action if there's no ID?

Reply:
Indeed. Fixed by just removing the link if there's no ID. I can't be sure if this is called with new actions anywhere now or in the future, so this seemed like the shortest and safest fix. Same with cohorts below as well.